proto/exocore/operator/v1/query.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+package exocore.operator.v1;
+
+import "cosmos_proto/cosmos.proto";
+import "exocore/operator/v1/tx.proto";
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "tendermint/crypto/keys.proto";
+
+option go_package = "github.com/ExocoreNetwork/exocore/x/operator/types";
+
+// QueryOperatorInfoReq is the request to obtain the operator information.
+message GetOperatorInfoReq {
+  // operator_addr is the operator address.
```

Comment:
To confirm, this is the `sdk.AccAddress` right? If yes, please add a comment to prevent confusion with traditional Cosmos SDK where the operator's address is generally of the type `sdk.ValAddress`.

Reply:
Yeah,  I will add a comment